lib/di/interceptor/app_interceptor.dart

```diff
@@ -26,27 +27,25 @@ class AppInterceptor extends Interceptor {
 
   @override
   void onError(DioError err, ErrorInterceptorHandler handler) {
-    final statusCode = err.response?.statusCode;
-    if ((statusCode == HttpStatus.forbidden ||
-            statusCode == HttpStatus.unauthorized) &&
-        _requireAuthentication) {
-      _doRefreshToken(err, handler);
-    } else {
-      handler.next(err);
-    }
+    handler.next(err);
+    // TODO: Integrate refresh-token https://github.com/nimblehq/ic-flutter-taher-toby/issues/20
+    // final statusCode = err.response?.statusCode;
+    // if ((statusCode == HttpStatus.forbidden ||
+    //         statusCode == HttpStatus.unauthorized) &&
+    //     _requireAuthentication) {
+    //   _doRefreshToken(err, handler);
+    // } else {
+    // }
```

Comment:
Why do we need to disable this, please?

Reply:
@luongvo Seems like it's going into an infinite loop of requests when the token is unauthorized

So I remove the refresh token function for now, so we can look into it deeper later

https://user-images.githubusercontent.com/18277915/229685785-bd872ea1-f1a2-4126-8f2c-592f16cb0ab1.mov


I'll update the insights 🙏